app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -338,7 +338,9 @@ public void onItemClick(int position) {
         @Override
         public boolean onItemLongClick(int position) {
             if (position < navAdapter.getFragmentTags().size()) {
-                DrawerPreferencesDialog.show(getContext(), () -> navAdapter.notifyDataSetChanged());
+                DrawerPreferencesDialog.show(getContext(), () -> {
+                    navAdapter.notifyDataSetChanged();
+                });
```

Comment:
Please revert this formatting change. That reduces the diff and makes it easier to use `git blame` later.